AlphaWallet/TokenScriptClient/Models/TokenScript.swift

```diff
@@ -14,8 +14,8 @@ extension TokenScript {
     static let repoClientName = "AlphaWallet"
     static let repoPlatformName = "iOS"
     static let tokenScriptNamespacePrefix = "http://tokenscript.org/"
-    static let oldNoLongerSupportedTokenScriptNamespaceVersions = ["2019/04"].map { "\(tokenScriptNamespacePrefix)\($0)/tokenscript" }
-    static let supportedTokenScriptNamespaceVersion = "2019/05"
+    static let oldNoLongerSupportedTokenScriptNamespaceVersions = ["2019/05"].map { "\(tokenScriptNamespacePrefix)\($0)/tokenscript" } + ["2019/04"].map { "\(tokenScriptNamespacePrefix)\($0)/tokenscript" }
```

Comment:
This can be:

```
static let oldNoLongerSupportedTokenScriptNamespaceVersions = ["2019/04", "2019/05"].map { "\(tokenScriptNamespacePrefix)\($0)/tokenscript" }
```